### ISSUE TITLE
Reusing results from `tune!`

### ISSUE THREAD

**Issue by jonas-schulze** [JuliaCI/BenchmarkTools.jl#182]
Are the measurements taken using `tune!(suite)` reused/incorporated in the min/mean/etc values from `run(suite)`?

**Comment by gdalle** [comment]
No, they are only used to select appropriate parameters. At least I think so
